perl.h

```diff
@@ -4470,7 +4470,16 @@ intrinsic function, see its documents for more details.
 #endif
 
 #if defined(__CYGWIN__) || defined(__MVS__)
+
 void init_os_extras(void);
+
+#  if defined(__MVS__)
+#    ifdef EBCDIC
+#      define __setfdccsid(to_fd,  bits)  0
+#    else
+#      include <zos.h>
+#    endif
+#  endif
```

Comment:
I think this could be moved outside the `#if defined(__CYGWIN__) || defined(__MVS__)`

Reply:
agreed